avocado/plugins/beaker_result.py

```diff
@@ -0,0 +1,137 @@
+"""
+beaker result plugin
+
+Sends results and logs to beaker using the harness API.
+https://beaker-project.org/docs/alternative-harnesses/index.html
+
+Finds the beaker API entry point using the BEAKER_LAB_CONTROLLER_URL
+environment variable.  Does nothing in case the variable is not set.
+
+"""
+import glob
+import os
+import pprint
+import urllib.error
+import urllib.parse
+import urllib.request
+
+from avocado.core.output import LOG_UI
+from avocado.core.plugin_interfaces import ResultEvents
+
+
+class BeakerResult(ResultEvents):
+
+    """
+    send test results to beaker test harness api
+    """
+
+    name = 'beaker'
+    description = 'report results to beaker'
+
+    beaker_url = None
+    job_id = None
+
+    def __init__(self, config):
+        self.config = config  # make lint happy
```

Comment:
This works, but it's preferable to do:

```python
    def __init__(self, config=None):  # pylint: disable=W0613
```

That avoids the empty assignment (and attribute).